### PR TITLE
Wysiwyg save fix

### PR DESCRIPTION
**Pull request by rmathew1011**

### CODE REVIEW

**Comment by wwelling** [comment]
@rmathew1011 has this been deployed in dev for vetting?

**Comment by rmathew1011** [comment]
> @rmathew1011 has this been deployed in dev for vetting? It would be easier to review to avoid the excessive time sink to build and deploy locally for testing.

Would wait for PR approval and then proced for DEV deployment request.
At present. there is a blocker regarding the `Paste` and `Print` plugins - have created issues for the same.

File: package.json [hunk 1]
```diff
@@ -76,7 +78,7 @@
     "tinymce": "6.2.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "12.2.18",
+    "@angular-devkit/build-angular": "12.2.16",
```
Comment (wwelling) [inline package.json:81]: What is the reason for the patch downgrade?

Comment (wwelling) [reply package.json:81]: Is it to match the other `@angular` packages or incompatibility?

File: projects/wvr-elements/src/lib/wvr-wysiwyg/wvr-wysiwyg.component.ts [hunk 2]
```diff
@@ -49,15 +49,15 @@ export class WvrWysiwygComponent extends WvrBaseComponent implements OnInit, OnD
     base_url: 'tinymce',
     skin: 'oxide',
     plugins: [
-      'advlist autolink lists link image charmap print',
-      'preview anchor searchreplace visualblocks code',
-      'fullscreen insertdatetime media table paste',
-      'help wordcount print preview save'
+      "advlist", "autolink", "lists", "link", "image", "charmap", "print",
```
Comment (wwelling) [inline projects/wvr-elements/src/lib/wvr-wysiwyg/wvr-wysiwyg.component.ts:52]: Should we be consistent with the use of single/double-quotes?

**Pull request merged**